app/helpers/posts_helper.rb

```diff
@@ -0,0 +1,12 @@
+module PostsHelper
```

Comment:
If the application were larger, you would probably want to use this method across multiple controllers. In that case, the best place for it would be in the `application_helper.rb` file. It works here for now, but keep this in mind for later.